gmail/1.0/modules/ROOT/pages/gmail-connector-reference.adoc

```diff
@@ -0,0 +1,5497 @@
+= Gmail Connector 1.0 Reference
+
+
```

Comment:
Add the connector description here. You can just use the same one you have in the index file.